ibm/resource_ibm_container_cluster.go

```diff
@@ -157,6 +157,16 @@ func resourceIBMContainerCluster() *schema.Resource {
 				ValidateFunc: validateWorkerNum,
 			},
 
+			"labels": {
+				Type:             schema.TypeMap,
+				Optional:         true,
+				Computed:         true,
+				ForceNew:         true,
```

Comment:
Does this mean that adding labels to the default cluster will cause it to be destroyed and recreated? If so, is that required?

Reply:
No required, the API support changes.